EmmyLua/FrameXML/BNet.lua

```diff
@@ -1,3 +1,65 @@
-WOW_PROJECT_MAINLINE = 1
-WOW_PROJECT_CLASSIC = 2
-WOW_PROJECT_BURNING_CRUSADE_CLASSIC = 5
+---@type number
+WOW_PROJECT_MAINLINE = 0
+
+---@type number
+WOW_PROJECT_CLASSIC = 0
+
+---@type number
+---Note: It's not yet available on Retail, Classic, PTR
+WOW_PROJECT_BURNING_CRUSADE_CLASSIC = 0
```

Comment:
Why are you assigning them as zero?

`WOW_PROJECT_ID` I can understand since that's different, but not for `WOW_PROJECT_MAINLINE`, and `WOW_PROJECT_CLASSIC`, etc

Reply:
- If you mark an object with `@type [type]` then it doesn't matter what value is assigned to it
- It would be good if variables of this kind (constants) were invariant. No matter what they contain. On the one hand, it would save us (contributors) from the need to track value changes. On the other hand, users should also not consider specific values when using them.
   Do you need a specific version of WoW? Compare `WOW_PROJECT_ID` to the desired version. Do you need to iterate over all chat tabs? Use `NUM_CHAT_WINDOWS` in a for loop. 
- The server is able to parse, analyze code (though not without problems), but can't interpret or execute it. It doesn't make sense to write something. It may even complicate its work. So I prefer declaring variables than assigning specific values to them

Reply:
* I think those shouldn't even be annotated / marked with `@type [type]`. I don't agree that it would be good to mark constants as just some invariant type. Otherwise there is no point in documenting them if users only know the type and now the actual value. Also for the point that the server can't interpret or execute code, is not really applicable for constants...